### PR TITLE
Solutions page: Software supply chain security 🚀

### LINKED ISSUE #3555: Create new solution page
URL: /solutions/supply-chain-security

[View mockup here](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/website?node-id=9362%3A23499&t=Q0cGtNp61FfWih9l-1)

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

:construction: Page is accessible at [`/solutions/supply-chain-security`](https://deploy-preview-3559--gitpod-kumquat.netlify.app/solutions/supply-chain-security)

## Related Issue(s)

Fixes #3555

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3559"></a>

### CODE REVIEW

File: src/lib/contents/home/features.ts [hunk 1]
```diff
@@ -269,3 +269,68 @@ export const onboardFeatures: Feature[] = [
...
+    height: "24%",
+  },
+};
+
+export const lockDownYourSourceCode: Feature = {
+  title: "Lock down your source code",
+  paragraph:
+    "Secure your source code by keeping it in one, centralised location that you control - the cloud.",
+  image: {
+    src: "/svg/icons/cloud.svg",
```
Comment (Siddhant-K-code) [inline src/lib/contents/home/features.ts:312]: This needs discussion (in [figma](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj?node-id=9362:23499#409180970))

File: src/lib/contents/home/features.ts [hunk 2]
```diff
@@ -269,3 +269,68 @@ export const onboardFeatures: Feature[] = [
...
+    alt: "All your data secured",
+    width: "33%",
+    height: "33%",
+  },
+};
+
+export const secureDataMovement: Feature = {
+  title: "Secure data movement",
+  paragraph:
+    "Gitpod is GDPR compliant, SOC 2 Type II compliant and provides clients with Data Processing Agreements (DPA) incorporating the Standard Contractual Clauses (SCC) for International Data Transfers.",
```
Comment (Siddhant-K-code) [inline src/lib/contents/home/features.ts:322]: Suggestion from @securitymirco - [here (internal)](https://gitpod.slack.com/archives/C01KGM9ETU6/p1680535842363629?thread_ts=1680512181.928759&cid=C01KGM9ETU6)

```suggestion
    "Gitpod is SOC 2 Type II compliant and conducts pentesting at least annually. The company is also GDPR compliant.",
```

**Comment by Siddhant-K-code** [comment]
cc: @taliamoyal (unable to add you as a reviewer, because you need to join `gitpod-io` GitHub Org. 🙏🏽)

**Review by chrifro — COMMENTED** [review]
Nice ✨

- [ ] I noticed one thing that affects all solution pages: when the banner is visible the top spacing is not automatically adjusted and too little

Bildschirm­foto 2023-04-06 um 12 54 40

- [x] for "all your data secured" can you use exactly the same image as in Figma? The letters should not be readable. Let me know if I can help with the export

- [x] Adjust the spacing for mobile (less on the top and more before the card)
 
Bildschirm­foto 2023-04-06 um 12 58 49

Feel to merge it afterward  🙌

**Pull request merged**